CONTRIBUTING.md

```diff
@@ -1,10 +1,6 @@
 # Contributing
 
-| Component            | Build Status                                                                                                                                                                                                          |
```

Comment:
Not sure why this was included in a contributing guide, but the badge stated "Never run" so it wasn't providing any value.

Reply:
It looks like there was once one for Client Libraries, too! https://github.com/Azure/azure-sdk-for-net/pull/7052/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R6